### PR TITLE
[fr] Dic addition from Ongoing feedback board issue

### PR DESCRIPTION
**Pull request by LucieSteib**
In the context of [fr] [Ongoing] Missing words in French dictionary
[#5240](https://github.com/languagetooler-gmbh/languagetool-premium/issues/5240)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206959311454975

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt [hunk 1]
```diff
@@ -5,4 +5,83 @@
...
+monoglycérides;monoglycéride;N m p
+tapinose;tapinose;N f s
+tapinoses;tapinose;N f p
+Pangloss;Pangloss;Z m s
+Toinette;Toinette;Z f s
+Diafoirus;Diafoirus;Z m s
+Mercutio;Mercutio;Z m s
+Tybalt;Tybalt;Z m s
+Missak Manouchian;Missak Manouchian;Z m s
+Mélinée Manouchian;Mélinée Manouchian;Z f s
```
Comment (jaumeortola) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt:33]: This won't work here. Add multi-token expressions to multiwords.txt

Comment (GillouLT) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt]: Thanks Jaume! Just fixed it, merging.

**Review by GillouLT — COMMENTED** [review]
LGTM

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt (hunk 1 above)
Comment (LucieSteib) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt]: Dear lord, thanks, it was definitely too late to work :/

**Pull request merged**